pytest/tests/loadtest/locust/common/base.py

```diff
@@ -252,7 +252,7 @@ def send_tx(self, tx: Transaction, locust_name) -> dict:
                 submit_response = submit_raw_response.json()
                 # extract transaction ID from response, it should be "{ "result": "id...." }"
                 if not "result" in submit_response:
-                    meta["exception"] = RpcError(submit_response,
+                    meta["exception"] = RpcError(details=submit_response,
```

Comment:
Let's swap the order in which arguments are passed to match that of the function signature (and other places where we call this constructor)

Reply:
I couldn't find other calls where we use both arguments, usually we only specify the message. I fixed this one and will merge it now.